test/parallel/test-cluster-concurrent-disconnect.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
this test is hanging

Reply:
Where can I see that?

Reply:
oh, sorry. I edit it on my local env so It's broken and I didn't notice that